gui/src/pages/NewProjectConfig.svelte

```diff
@@ -88,24 +91,11 @@
         {#await dirPromise}
           <Spinner />
         {:then dir}
-          {#if dir.parent !== null}
-            <button
-              on:click={() => setWorkingDirectory(String(dir.parent?.path))}
-            >
-              ..
-            </button>
-          {:else}
-            <button disabled> .. </button>
-          {/if}
-          <div class="directories">
-            {#each dir.entries
-              .filter((x) => x.isDirectory)
-              .sort((x, y) => x.name.localeCompare(y.name)) as entry}
-              <button on:click={() => setWorkingDirectory(entry.path)}
-                >{entry.name}</button
-              >
-            {/each}
-          </div>
+          <DirectoryBrowser
```

Comment:
yes, more smaller components all the time :)

Also, I still strongly recommend you start making more aggressive use of Storybook :)